.terraform.docs.yml

```diff
@@ -0,0 +1,22 @@
+formatter: "md"
+
+content: |-
```

Comment:
I expect this content section can be removed as this is the default behaviour

Reply:
removed